src/app/forgot-password/forgot-password.component.html

```diff
@@ -19,18 +19,30 @@ <h4 class="mt-5 font-weight-bold">Forgot Password?</h4>
                 label="Email id"
                 [control]="emailControl"
                 [isEmail]="true"
-                [errorMessage]="emailIncorrect"
                 class="form-group mb-3"
               ></app-base-input>
 
+              <div
+                *ngIf="emailControl.hasError('pattern') && emailControl.touched"
+                class="text-danger"
+              >
+                Please enter a valid email address.
```

Comment:
this message should be come from constants file.

in constants file you can add all the strings which you need in future as well.